app/assets/javascripts/misc/previous-address.js

```diff
@@ -0,0 +1,37 @@
+import 'classlist.js';
+
+const I18n = window.LoginGov.I18n;
+
+function previousAddress() {
+  const accordion = document.querySelector('.accordion');
+
+  if (accordion) {
+    const controls = accordion.querySelector('[aria-controls]');
+    const selects = accordion.querySelectorAll('select');
+    const inputs = accordion.querySelectorAll('input');
+
+    controls.classList.remove('display-none');
```

Comment:
I did this for the non-js state - without it, it would look like this:

![screen shot 2017-02-15 at 10 07 18 am](https://cloud.githubusercontent.com/assets/1178494/22980178/c2ae0312-f366-11e6-94be-2b6f610553f9.png)

I didn't want the "Add previous address" displayed.  Maybe there is a better way to handle this though?